app/config.py

```diff
@@ -340,6 +339,12 @@ class Config(object):
                 'schedule': crontab(hour=4, minute=0),
                 'options': {'queue': QueueNames.PERIODIC},
             },
+            'send-scheduled-comp-and-pen-sms': {
+                'task': 'send-scheduled-comp-and-pen-sms',
+                # At every minute past every hour from 13 through 21 on every day-of-month from 21 through 28
+                'schedule': crontab(hour='13-21', day_of_month='21-28'),
```

Comment:
```suggestion
                'schedule': crontab(hour='13-21', day_of_month='21-31'),
```
In the event it takes us until the end of the month.
https://docs.celeryq.dev/en/stable/reference/celery.schedules.html#celery.schedules.crontab.day_of_month

Reply:
What if the month only has 28 days? Is it going to throw an error if it tries to run on a day that doesn't exist for the month? Or it just silently fails?

Or do we need to do something tricky and have a line for months that have 30 days, a line for months with 31 days, a line for feb which has 28 days (then what happens in leap year?)

Reply:
When I looked it up, it says it just doesn't run if the day doesn't exist in the month, so it should work fine like that.

Reply:
Their documentation should be more clear. 

I poked around their code before suggesting the change and I'm confident it won't have trouble with 31. This is the month to check! 😆 The [cron(5)](https://www.man7.org/linux/man-pages/man5/crontab.5.html) docs that crontab is based on say it only triggers on matches. So essentially it wouldn't matter because it's just a max value. 

>Commands are
       executed by [cron(8)](https://www.man7.org/linux/man-pages/man8/cron.8.html) when the 'minute', 'hour', and 'month of the
       year' fields match the current time, and at least one of the two
       'day' fields ('day of month', or 'day of week') match the current
       time (see "Note" below).